EdgeModuleMessageSample/Parameters.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    public enum Transport
+    {
+        Mqtt,
+        Amqp
+    };
+
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'c',
+            "PrimaryConnectionString",
+            Required = true,
+            HelpText = "The primary connection string for the module to simulate.")]
+        public string PrimaryConnectionString { get; set; }
+
+        [Option(
+            't',
+            "Transport",
+            Default = Transport.Mqtt,
+            Required = false,
+            HelpText = "The transport to use for the connection.")]
+        public Transport Transport { get; set; }
+
+        [Option(
+           "TransportProtocol",
+           Default = IotHubClientTransportProtocol.Tcp,
+           HelpText = "The transport to use to communicate with the device provisioning instance.")]
+        public IotHubClientTransportProtocol TransportProtocol { get; set; }
+
+        [Option(
+            'o',
+            Default ="*",
+            Required = false,
+            HelpText = "Output target of the module receiving telemetry message.")]
+        public string OutputTarget { get; set; }
```

Comment:
Why is this a parameter?

Reply:
for previews/v2, output target is required for sending telemetry from module. Default value is '*'

Reply:
Right, but why make it parameter rather than just hardcode it. How do we expect the user to know what to do with it as a parameter and what will change with the sample if they pass something else?